src/Core/Sts/Result/AssumeRoleResponse.php

```diff
@@ -8,8 +8,23 @@
 
 class AssumeRoleResponse extends Result
 {
+    /**
+     * The temporary security credentials, which include an access key ID, a secret access key, and a security (or session)
+     * token.
+     */
     private $Credentials;
+    /**
```

Comment:
Oh no, I dont like this CS anymore =)

Can you remove this line from .php.cs: `'class_attributes_separation' => ['elements'=>['method']],`?

That will add an extra space between properties =)